connectionbuddy/src/main/java/com/zplesac/connectionbuddy/models/ConnectivityEvent.java

```diff
@@ -3,48 +3,66 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 
+import androidx.annotation.NonNull;
+
 /**
  * Created by Željko Plesac on 23/09/15.
  * Connectivity event which holds all the necessary data about network connection.
  */
 public class ConnectivityEvent implements Parcelable {
 
+    @NonNull
     private ConnectivityState state;
 
+    @NonNull
     private ConnectivityType type;
 
+    @NonNull
     private ConnectivityStrength strength;
 
+    /**
+     * Initializes the object with DISCONNECTED state, NONE type, and UNDEFINED strength.
+     */
     public ConnectivityEvent() {
```

Comment:
We should also write this in the documentation, so that people will know what's the default event.

Reply:
What do you mean? This comment is a docstring. This constructor overload is not used anywhere in the library code. It's called only if the user calls it themselves, and if they decide to do so, the docstring will be displayed when using autocomplete.

Reply:
I've added this to `Readme.md`.

Reply:
Tnx.